salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume-cleanup

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+import argparse
+import errno
+import os
+import stat
+import sys
+
+
+SPARSE_FILES_DIR = '/var/lib/metalk8s/storage/sparse/'
+LOOP_MAJOR = 7
```

Comment:
You could cite [Documentation/admin-guide/devices.txt](https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/devices.txt#L191) from linux source tree as reference.